src/sentry/utils/snuba.py

```diff
@@ -70,14 +70,19 @@
 }
 
 
-DATASETS = {Dataset.Events: SENTRY_SNUBA_MAP, Dataset.Discover: DISCOVER_COLUMN_MAP}
+DATASETS = {
+    Dataset.Events: SENTRY_SNUBA_MAP,
+    Dataset.Discover: DISCOVER_COLUMN_MAP,
+    Dataset.Transactions: SENTRY_SNUBA_MAP,
+}
 
 # Store the internal field names to save work later on.
 # Add `group_id` to the events dataset list as we don't want to publically
 # expose that field, but it is used by eventstore and other internals.
 DATASET_FIELDS = {
     Dataset.Events: list(SENTRY_SNUBA_MAP.values()),
     Dataset.Discover: list(DISCOVER_COLUMN_MAP.values()),
+    Dataset.Transactions: list(SENTRY_SNUBA_MAP.values()),
```

Comment:
Are you sure this is correct? The `sentry_snuba_map` uses the event_name aliases which are not always the same as the ones on transactions. For example, the user columns and release columns have different names. You may need to go back and add the transaction aliases to snuba.events.Columns. I removed them a few months ago when we stopped using the transactions dataset for discover.